src/acp/v1.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  hasLinkedAcr,
+  addAcrPolicyUrl,
+  addMemberAcrPolicyUrl,
+  getAcrPolicyUrlAll,
+  getMemberAcrPolicyUrlAll,
+  removeAcrPolicyUrl,
+  removeAcrPolicyUrlAll,
+  removeMemberAcrPolicyUrl,
+  removeMemberAcrPolicyUrlAll,
+  Control,
+} from "./control";
+import {
+  internal_addMemberPolicyUrl,
+  internal_addPolicyUrl,
+  internal_createControl,
+  internal_getAcr,
+  internal_getControl,
+  internal_getControlAll,
+  internal_getMemberPolicyUrlAll,
+  internal_getPolicyUrlAll,
+  internal_removeMemberPolicyUrlAll,
+  internal_removePolicyUrl,
+  internal_removePolicyUrlAll,
+  internal_setAcr,
+  internal_setControl,
+} from "./control.internal";
+import { removeThing } from "../thing/thing";
+
+const v1ControlFunctions = {
+  hasLinkedAcr,
+  addAcrPolicyUrl,
+  addMemberAcrPolicyUrl,
+  getAcrPolicyUrlAll,
+  getMemberAcrPolicyUrlAll,
+  removeAcrPolicyUrl,
+  removeAcrPolicyUrlAll,
+  removeMemberAcrPolicyUrl,
+  removeMemberAcrPolicyUrlAll,
+};
+const deprecatedFunctions = {
+  createControl: internal_createControl,
+  getControl: internal_getControl,
+  getAllControl: internal_getControlAll,
+  setControl: internal_setControl,
+  removeControl: removeControl,
+  addPolicyUrl: internal_addPolicyUrl,
+  getPolicyUrlAll: internal_getPolicyUrlAll,
+  removePolicyUrl: internal_removePolicyUrl,
+  removePolicyUrlAll: internal_removePolicyUrlAll,
+  addMemberPolicyUrl: internal_addMemberPolicyUrl,
+  getMemberPolicyUrlAll: internal_getMemberPolicyUrlAll,
+  removeMemberPolicyUrl: internal_getMemberPolicyUrlAll,
+  removeMemberPolicyUrlAll: internal_removeMemberPolicyUrlAll,
+};
+
+/**
+ * @hidden
+ * @deprecated Replaced by [[acp_v2]].
+ */
+export const acp_v1 = {
+  ...acpAcp,
+  ...acpPolicy,
+  ...acpRule,
+  ...acpMock,
+  ...v1ControlFunctions,
+  ...deprecatedFunctions,
+};
+
+/**
+ * ```{note} The Web Access Control specification is not yet finalised. As such, this
+ * function is still experimental and subject to change, even in a non-major release.
+ * ```
+ *
+ * Remove an [[Control]] from the [[AccessControlResource]] of a Resource.
+ *
+ * @param withAccessControlResource A Resource with the Access Control Resource from which to remove an Access Control.
+ * @param control The [[Control]] to remove from the given Access Control Resource.
+ * @returns The given Resource with a new Access Control Resource equal to the original Access Control Resource, excluding the given Access Control.
+ * @hidden Developers don't need to care about initialising Controls - they can just add Policies directly.
+ * @deprecated
+ */
+export function removeControl<ResourceExt extends acpAcp.WithAccessibleAcr>(
+  withAccessControlResource: ResourceExt,
+  control: Control
+): ResourceExt {
+  const acr = internal_getAcr(withAccessControlResource);
+  const updatedAcr = removeThing(acr, control);
+  const updatedResource = internal_setAcr(
+    withAccessControlResource,
+    updatedAcr
+  );
+  return updatedResource;
+}
```

Comment:
Why isn't this function in `control.internal.ts` ?

Reply:
It's the only one the newer API's do not use internally. (I was doubting whether I wanted to re-use the other ones in the first place, since they're so trivial and their names overlap, or if I should just move all old functions into `v1.ts`. I'm still not sure about that :P )